index.html

```diff
@@ -4,164 +4,188 @@
-    <link rel="stylesheet" href="./assets/css/index.scss">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/css/main.css">
   </head>
   <body>
-    <header>
-      <nav>
-        <div class="">
-          <img src="https://www.google.com/url?q=http://www.pwcgov.org/government/dept/cs/PublishingImages/get%2520on%2520track.jpg&sa=D&source=hangouts&ust=1553871507546000&usg=AFQjCNEbzETM8hn2b6ZS_9UdKKc-dBhIEQ" alt=""></li>
-        </div>
-        <ul>
-          <li>Service</li>
-          <li>Shop</li>
-          <li>Events</li>
-          <li>Comunity</li>
+    <header class="container">
+      <div class="navbar">
+        <a href="#" class="logo-href"><img src="./assets/image/logo.jpg" alt="" class="img-logo"></a>
+        <ul class="nav-ul">
+            <li class="nav-item"><a class="nav-link" href="#">Home</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">About</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Service</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Poduct</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Events</a></li>
+            <li class="nav-item mr-4"><a class="nav-link" href="#">community</a></li>
         </ul>
-      </nav>
+        <div class="hamburger">
+          <i class="fas fa-bars hamburger-icon text-center"></i>
+        </div>
+      </div>
     </header>
-    <main>
-      <div class="">
-        <div id="demo" class="carousel slide" data-ride="carousel">
-          <!-- Indicators -->
-          <ul class="carousel-indicators">
-            <li data-target="#demo" data-slide-to="0" class="active"></li>
-            <li data-target="#demo" data-slide-to="1"></li>
-            <li data-target="#demo" data-slide-to="2"></li>
-          </ul>
-          <!-- The slideshow -->
-          <div class="carousel-inner">
-            <div class="carousel-item active">
-              <img src="la.jpg" alt="Los Angeles">
-            </div>
-            <div class="carousel-item">
-              <img src="chicago.jpg" alt="Chicago">
-            </div>
-            <div class="carousel-item">
-              <img src="ny.jpg" alt="New York">
-            </div>
-          </div>
-          <!-- Left and right controls -->
-          <a class="carousel-control-prev" href="#demo" data-slide="prev">
-            <span class="carousel-control-prev-icon"></span>
-          </a>
-          <a class="carousel-control-next" href="#demo" data-slide="next">
-            <span class="carousel-control-next-icon"></span>
-          </a>
+    <main class="container">
+      <div class="contain">
+        <div class="container">
+          <img class="mySlides" src="./assets/image/bg1.jpg" style="width:100%" alt="">
```

Comment:
why inline css?is not it possible to include it in the framework? and please do not left alt tag empty  from now on.

Reply:
changed that part too.